contents/docs/self-host/deploy/upgrade-notes.md

```diff
@@ -132,3 +132,25 @@ If you didn’t make any customization to those, there’s nothing you need to d
     - `clickhouse.persistentVolumeClaim` -> `clickhouse.persistence.existingClaim`
 
     If you are overriding any of those, please make the corresponding changes before upgrading. Depending on your settings and setup, during this upgrade the ClickHouse pod might get recreated.
+
+### Upgrading from 10.x.x
+
+11.0.0 removes some legacy Helm annotations that are not necessary anymore (see [#179](https://github.com/PostHog/charts-clickhouse/pull/179) for more info).
```

Comment:
Nit: Would be better to tell users what they get out of it. E.g. removing these annotations means upgrading doesn't cause downtime.

Reply:
I've updated the text to include your suggestion 🙌